.github/workflows/auto_update_doc.yml

```diff
@@ -37,14 +37,17 @@ jobs:
         
     - name: Install ONNX from source and update documentation
       run: |
+        source workflow_scripts/protobuf/build_protobuf_unix.sh 3
         python -m pip install -q --upgrade pip
         python -m pip install -r requirements-release.txt
         git submodule update --init --recursive
-        
-        source workflow_scripts/protobuf/build_protobuf_unix.sh "4" "$(pwd)"/protobuf/protobuf_install
-        
+              
         export ONNX_ML=1
-        pip install -v -e .
+
+        # Remove protobuf specification from requirements.txt to
+        # prevent pip from installing the latest version.
+        sed -i -e '/protobuf/d' requirements.txt
+        pip install --verbose -e .
```

Comment:
This is a little odd?

Reply:
I commented above the command. The idea is that pyproject.toml specifies requirements.txt, and it contains "protobuf>=4.25.1". This requirement updated protobuf to latest version after installation of ONNX in my Docker container. The easiest fix is to remove that sentence before pip install.

Reply:
I see. For my understanding, why do we want to fix the protobuf version to 4.25.1? I also wonder if installing protobuf==4.25.1 before running `pip install --verbose -e .` will work just fine

Reply:
In this case we want to test the version in requirements-release.txt.

Reply:
This workflow updates the documentation so I think it should work with 5.x as well? In any case I am not very comfortable having the workflow change the requirements file. If it’s ok with you we can merge the rest of the changes in this PR and create a follow up to address this workflow.

Reply:
I commented out the line containing the sed command